### DISCUSSION TITLE
Random new error on launch

### DISCUSSION THREAD

**Discussion by Josh65-2201** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#419] [Q&A] [answered]
I have this error appear on launch every time and ignoring it brings up the set up token page. the submit button doesn't work and relaunching or installing didn't do anything
![image](https://user-images.githubusercontent.com/26977075/121260256-fe36d780-c8a8-11eb-99a8-73817786454a.png)

**Comment by sumeshir26** [comment, marked as answer]
@josh-65 You can find the solution in the [FAQs](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#why-does-the-get-token-screen-fail-saving-tokens).

**Reply by Josh65-2201** [reply]
This is not a duplicate at all. Mine doesn't work at all, showing that error in the screenshot then "enter your save token code" witch the submit button does work. There's does work and this has nothing to do with web previews not showing up.

**Reply by sumeshir26** [reply]
@josh-65 Sorry, linked the wrong issue, will edit.

**Reply by Josh65-2201** [reply]
Ok thanks
